.github/workflows/ci.yml

```diff
@@ -36,3 +36,4 @@ jobs:
         run: npm ci
 
       - run: npm test
+      - run: grunt deploy-build
```

Comment:
Can we avoid to re-build the item (JS/CSS/Assets) that was already built via the "npm run prepare" script?

Reply:
@duboisp Done. I have removed the tasks that are already executed in prepare from "deploy-build" since this task is only run in our CI/CD and is always prepended by the prepare task.